tests/codechecker_config.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024, Basalte bv
```

Comment:
i would put this file somewhere else, the config might be applied when building tests, but we are checking the zephyr code, not the tests only, maybe a top level dot file or directory with multiple configs?